target/checkstyle-cachefile

```diff
@@ -0,0 +1,11 @@
+#Mon Aug 09 16:48:04 MSK 2021
+C\:\\Users\\Alehandro\\IdeaProjects\\mavenhw1\\src\\test\\java\\tc\\MultiplyTest.java=1628168948714
+configuration*?=60E5CA1B78B005BD7C50A6E6BB32E3B18EDC2367
+C\:\\Users\\Alehandro\\IdeaProjects\\mavenhw1\\src\\main\\java\\com\\epam\\tc\\Operations.java=1628099677182
+C\:\\Users\\Alehandro\\IdeaProjects\\mavenhw1\\src\\test\\java\\tc\\DataProviders.java=1628168991503
+module-resource*?\:checkstyle-xpath-suppressions.xml=DC9D28C8998BED753DA591942D51B5838B7CC670

```

Comment:
`target` folder should not be included into git

Reply:
ok